src/libs/ajax/leonardo/Runtimes.test.js

```diff
@@ -45,4 +46,21 @@ describe('Runtimes ajax', () => {
       expect(mockFetchLeo).toHaveBeenCalledWith(`api/google/v1/runtimes/${runtime.googleProject}/${runtime.runtimeName}/start`, expect.anything())
     }
   })
+
+  // TODO (LM) failing on TypeError: (0 , _ajaxCommon.jsonBody) is not a function
```

Comment:
is this TODO something to worry about?

Reply:
oh lol I forgot to get rid of that comment, will do